contents/product-engineers/nps-vs-csat-vs-ces.md

```diff
@@ -0,0 +1,202 @@
+
+For example, let's say you survey `100` people. `70` people give a positive score, your CSAT would be '70 / 100 = 70%`.
+
+Naturally, a higher CSAT indicates that users are likely to continue using your product. Conversely, low CSAT scores correlate with high churn rates.
+
+### What is a good CSAT score for SaaS?
+
+A good score is typically 75-85%. Anything higher is amazing, and anything lower is poor. 
+
+According to [fullview](https://www.fullview.io/blog/csat-benchmarks-by-industry#toc-csat-benchmarks-for-software-companies), the average CSAT score for SaaS apps is 78.
+
+### Drawbacks of CSAT
+
+Since CSAT survey questions are specific to a feature, customers only answer about what's being asked and nothing more. This means you may miss broader issues with your product.
+
+It's also important to note that a high CSAT score doesn't necessarily translate to user loyalty. Users might be satisfied with certain aspects of your product but could still switch to a competitor for reasons not covered in the survey – like pricing or other features. 
+
+Thus, while CSAT is useful for understanding short-term sentiment, it should be paired with other feedback methods to understand the long-term.
+
+### When to use CSAT surveys
+
+CSAT surveys are most effective when you need real-time feedback after specific interactions with your product.
+
+With this in mind, try to use CSAT surveys after key interactions in your user journey. For example:
+
+- **After onboarding –** to understand if your onboarding was effective and engaging.
+- **During [beta testing of a new feature](https://posthog.com/tutorials/beta-feedback) –**  to see if it meets your users' expectations.
+- **After major app updates –** to understand whether your UX has improved or if there are new issues to address.
+- **Before subscription renewal –** to allow you to address any issues beforehand.
+- **After customer support interaction –** to see if customers feel their issue has been resolved.
+
+## What is CES?
+
+Customer effort score (CES) measures how easy it is to use your product or feature. 
+
+Users are asked "How easy was it to use `<name of feature>`" and a response scale ranging from 1 (very difficult) to 5 (very easy).
+
+![CES survey](../images/blog/nps-csat-ces/ces-survey.png)
+
+CES is great for debugging UX problems in your app and pinpointing problem areas.
+
+### How to calculate CES score
+
+To find your CES score, you calculate the average score of all your responses. That is: `sum of all responses / number of responses = CES score`
+
+![CES score](../images/blog/nps-csat-ces/ces-score.png)
+
+For example, let's say you survey `100` people. If the sum of all their responses is `450`, your CES score is `4.5`.
+
+### What is a good CES score for SaaS?
+
+While there are no published benchmarks for CES scores, you should aim for a score of 4 or higher. Any lower is a sign that users are struggling to use your app and this will lead to churn.
```

Comment:
There seems to be some benchmarks? Like this gartner one: 

![Screenshot 2023-11-17 at 9 18 41 AM](https://github.com/PostHog/posthog.com/assets/34755028/744c674b-cc6b-44f2-8e9c-e21dcbc78d9d)

Reply:
This was the only one I was able to find. Most other articles are pointing to this, but the report is behind a paywall (and so I assume most are using this same picture as a result). I'm reluctant to include it since it doesn't give anymore context, like industry etc.